### PR TITLE
Fix(Data): Removed Spiribird's Call skill timer

### PR DESCRIPTION
**Pull request by WetU**
Removed Spiribird's Call
Fixed numeric type for armorskill Id;

### CODE REVIEW

**Comment by Haato3o** [comment]
Honestly, Spiribird's Call isn't even an abnormality, I don't know why it was added in the first place

**Comment by WetU** [comment]
Removed that skill from the list;

File: HunterPie.Integrations/Datasources/MonsterHunterRise/Definitions/MHREquipmentSkillStructure.cs [hunk 1]
```diff
@@ -5,6 +5,6 @@ namespace HunterPie.Integrations.Datasources.MonsterHunterRise.Definitions;
 [StructLayout(LayoutKind.Explicit)]
 public struct MHREquipmentSkillStructure
 {
-    [FieldOffset(0x10)] public int Id;
-    [FieldOffset(0x14)] public int Level;
+    [FieldOffset(0x10)] public byte Id;
```
[the remark this answers is not in the document]
Comment (WetU) [reply HunterPie.Integrations/Datasources/MonsterHunterRise/Definitions/MHREquipmentSkillStructure.cs:8]: Valid EquipSkillID is 1~255.
Its enum had 0(None) to 255(Max) EquipSkillIDs.

[the remark this answers is not in the document]
Comment (Haato3o) [reply HunterPie.Integrations/Datasources/MonsterHunterRise/Definitions/MHREquipmentSkillStructure.cs:8]: Enums in C# are integers by default, regardless the value range.

**Review by Haato3o — APPROVED** [review]

**Pull request closed without merging**